crates/dojo-lang/src/semantics/test_data/get

```diff
@@ -0,0 +1,408 @@
+                StatementLet {
+                    pattern: Variable(
+                        __Health_model_span__,
+                    ),
+                    expr: FunctionCall(
+                        ExprFunctionCall {
+                            function: ?12::span,
+                            args: [
+                                Value(
+                                    Snapshot(
+                                        ExprSnapshot {
+                                            inner: Var(
+                                                LocalVarId(test::__Health_model__),
+                                            ),
+                                            ty: @core::array::Array::<core::felt252>,
+                                        },
+                                    ),
+                                ),
+                            ],
+                            ty: core::array::Span::<?10>,
+                        },
+                    ),
+                },
+            ),
+            Let(
+                StatementLet {
+                    pattern: Variable(
+                        __Health,
+                    ),
+                    expr: FunctionCall(
+                        ExprFunctionCall {
+                            function: ?13::expect,
+                            args: [
+                                Value(
+                                    FunctionCall(
+                                        ExprFunctionCall {
+                                            function: ?14::deserialize,
+                                            args: [
+                                                Reference(
+                                                    LocalVarId(test::__Health_model_span__),
+                                                ),
+                                            ],
+                                            ty: core::option::Option::<<missing>>,
+                                        },
+                                    ),
+                                ),
+                                Value(
+                                    Literal(
+                                        ExprLiteral {
+                                            value: 7624190543980043505746182176517533098884860453715398488377751337573,
+                                            ty: ?12,
+                                        },
+                                    ),
+                                ),
+                            ],
+                            ty: ?11,
+                        },
+                    ),
+                },
+            ),
+        ],
+        tail: Some(
+            Var(
+                LocalVarId(test::__Health),
+            ),
+        ),
+        ty: <missing>,
+    },
+)
+
+//! > semantic_diagnostics
+error: Identifier not found.
+ --> get_inline_macro:3:26
+serde::Serde::serialize(@key, ref __get_macro_keys__);
```

Comment:
If we add key and world to setup and intern dojo-core in the semantic db i think these diagnostics will go away

Reply:
Aah, I misunderstood...
Yeah, I can look into it.

Reply:
To get rid of all diagnostic, we actually need the whole dojo crate to have it work.
Or mock stuff we need.

Reply:
Done
# 🫡